x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/action_connectors/utils/index.ts

```diff
@@ -40,6 +40,7 @@ export const handleActionsHaveNoConnectors = (
         : 'connector is missing. Connector id missing is:';
     errors.push(
       createBulkErrorObject({
+        id: actionsIds.join(),
```

Comment:
As `id` field stores a serialized array it worth to have a comment in `BulkError` and `ImportResponseError`  to highlight that fact it can contain more than one identifier in a case of missing connectors.

Reply:
We'll go ahead and add a comment there. Looks like `rule_id` has been used this way for a bit. Will be sure to document.

Reply:
Please review [bc5547c](https://github.com/elastic/kibana/pull/152313/commits/bc5547c9a5e562b986aec7e6d4e3d7bd4f28b6fb)